data/brands/shop/kiosk.json

```
@@ -232,6 +232,23 @@
         "name:ja": "キヨスク",
         "shop": "kiosk"
       }
+    },
+    {
+      "displayName": "ローソンプラストークス",
+      "id": "lawsontoks-84165a",
+      "locationSet": {"include": ["jp"]},
+      "matchNames": ["ローソン+トークス"],
+      "note": "LAWSON+toks has two types of stores: convenience store and kiosk.",
+      "tags": {
+        "brand": "LAWSON+toks",
+        "brand:en": "LAWSON+toks",
+        "brand:ja": "ローソンプラストークス",
+        "brand:wikidata": "Q115868119",
+        "name": "ローソンプラストークス",
```

Same as the convenience store version, I thought this one might be better.
```suggestion
      "note": "LAWSON+toks has two types of stores: convenience store and kiosk.",
      "tags": {
        "brand": "LAWSON+toks",
        "brand:en": "LAWSON+toks",
        "brand:ja": "ローソンプラストークス",
        "brand:wikidata": "Q115868119",
        "name": "ローソン+トークス",
```